servers/lumen/routes/web.php

```diff
@@ -180,15 +180,15 @@
 
     // WEIGHT
     $router->get('weight/v20/', ['uses' => 'WeightController@getAllTemporalWeights']);
-    $router->get('weight/v20/{serverSideRowUuid}', ['uses' => 'WeightController@getOneWeight']);
+    $router->get('weight/v20/{pServerSideRowUuid}', ['uses' => 'WeightController@getOneWeight']);
```

Comment:
p before a variable name indicates that it is a function parameter. Why have you use p over here?

Reply:
Sir, Routing variable name and function parameter variable name need to be same, otherwise I have got error. ->
"Unable to resolve dependency [Parameter #0 [ <required> $pServerSideRowUuid ]] in class App\Http\Controllers\WeightController"